pkg/drawio/layout.go

```diff
@@ -96,13 +106,156 @@ func canShareCell(i1, i2 IconTreeNodeInterface) bool {
 	return true
 }
 
+func sortBySize(groups []SquareTreeNodeInterface) []SquareTreeNodeInterface {
+	sortedBySizeGroups := make([]SquareTreeNodeInterface, len(groups))
+	copy(sortedBySizeGroups, groups)
+	sort.Slice(sortedBySizeGroups, func(i, j int) bool {
+		return len(sortedBySizeGroups[i].(*GroupSquareTreeNode).groupies) > len(sortedBySizeGroups[j].(*GroupSquareTreeNode).groupies)
```

Comment:
This assumes all members of the input slice are of type GroupSquareTreeNode.
If not checking this explicitly, at least add a comment with this assumption.

Reply:
done